hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -2512,47 +2513,121 @@ def hpat_pandas_series_median_impl(self, axis=None, skipna=True, level=None, num
     return hpat_pandas_series_median_impl
 
 
-@overload_method(SeriesType, 'dropna')
```

Comment:
Sorry, I don't understand. You renamed series.dropna to series.fillna? What happend with series.dropna?

Reply:
That was an automatic merge issue, sorry, fixed it.